api/lib/models/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var Sequelize = require('sequelize');
+var logger = require('../log/logger');
+var sequelizeURL = process.env.DB_URI;
+var sequelize = new Sequelize(sequelizeURL);
```

Comment:
It sets the pool to true but doesn't set up any values for the pool size, timeout, etc... I will add a story to track this.